apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -206,29 +207,39 @@ fn send_key(device: &mut Device, key: keyboard::Key) -> Result<(), CommandError>
 fn handle_command(device: &mut Device, data: &[u8]) -> Result<(), CommandError> {
     let ButtonStatusCommand { button, action } = data.try_into()?;
 
-    let key: keyboard::Key;
+    let key: Keyboard;
     match action {
         Action::Pressed => match button {
             Button::Select => {
-                key = keyboard::Key::Enter;
+                key = uinput::event::Keyboard::Key(keyboard::Key::Enter);
             }
```

Comment:
Use the `Keyboard` that you brought in scope above here and the rest of this `match`?
```suggestion
                key = Keyboard::Key(keyboard::Key::Enter);
```

Reply:
Since we settled on common keyboard keys we don't need `Keyboard::Misc` any more and simplified the types in https://github.com/votingworks/vxsuite/pull/4501/commits/e1a104bec3a4ac6e5cd13463ee572eb4578678dc